### PR TITLE
update translation of slice

### PR DESCRIPTION
**Pull request by yangmeijuan**
### Description
@JasonLamv-t .I have update the translation of pop.I modified and translated it according to the English document

### Related issues and pull requests
 [[zh-cn] The proofreading and update plan of Javascript.Array #12761](https://github.com/mdn/translated-content/issues/12761)

### CODE REVIEW

**Comment by JasonLamv-t** [comment]
Thanks for your contribution! I already add this PR to the list.

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
 ---
 title: Array.prototype.slice()
 slug: Web/JavaScript/Reference/Global_Objects/Array/slice
+page-type: javascript-instance-method
+browser-compat: javascript.builtins.Array.slice
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:5]: 根据[指南](https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md#%E5%85%83%E6%95%B0%E6%8D%AE)，这部份不需要添加到翻译中
```suggestion
```

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 2]
```diff
@@ -19,24 +21,29 @@ slice(start, end)
 
 ## 参数
 
-- `begin` {{optional_inline}}
-  - : 提取起始处的索引（从 `0` 开始），从该索引开始提取原数组元素。如果该参数为负数，则表示从原数组中的倒数第几个元素开始提取，`slice(-2)` 表示提取原数组中的倒数第二个元素到最后一个元素（包含最后一个元素）。如果省略 `begin`，则 `slice` 从索引 `0` 开始。如果 `begin` 超出原数组的索引范围，则会返回空数组。
+- `start` {{optional_inline}}
+  - : 提取起始处的索引（从 `0` 开始）, [转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#integer_conversion)。
+    - 如果索引是负数，则从数组末尾开始计算 —— 如果 `start < 0`，则使用 `start + array.length`。
+    - 如果 `start < -array.length` 或者省略了 `start`，则使用 `0`。
+    - 如果 `start >= array.length`，则不提取任何元素。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:28]:
```suggestion
- `start` {{optional_inline}}
  - : 提取起始处的索引（从 `0` 开始）, 会[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换)。
    - 如果索引是负数，则从数组末尾开始计算——如果 `start < 0`，则使用 `start + array.length`。
    - 如果 `start < -array.length` 或者省略了 `start`，则使用 `0`。
    - 如果 `start >= array.length`，则不提取任何元素。
```

**Comment by yangmeijuan** [comment]
> Thanks for your contribution! I already add this PR to the list.

Thank you. But I translated slice instead of splice

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 3]
```diff
@@ -19,24 +21,29 @@ slice(start, end)
...
 - `end` {{optional_inline}}
-  - : 提取终止处的索引（从 `0` 开始），在该索引处结束提取原数组元素。`slice` 会提取原数组中索引从 `begin` 到 `end` 的所有元素（包含 `begin`，但不包含 `end`）。`slice(1,4)` 会提取原数组中从第二个元素开始一直到第四个元素的所有元素（索引为 1, 2, 3 的元素）。如果该参数为负数，则它表示在原数组中的倒数第几个元素结束抽取。 `slice(-2,-1)` 表示抽取了原数组中的倒数第二个元素到最后一个元素（不包含最后一个元素，也就是只有倒数第二个元素）。如果 `end` 被省略，则 `slice` 会一直提取到原数组末尾。如果 `end` 大于数组的长度，`slice` 也会一直提取到原数组末尾。
+  - : 提取终止处的索引（从 `0` 开始）, [转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#integer_conversion). `slice()` 会提取到不包括 `end` 的位置。
+    - 如果索引是负数，则从数组末尾开始计算 —— 如果 `end < 0`，则使用 `end + array.length`。
+    - 如果 `end < -array.length` 或者省略了 `end`，则使用 `0`。
+    - 如果 `end >= array.length` 或者省略了 `end`，则使用 `array.length`，提取所有元素直到末尾。
+    - 如果 `end` 在标准化后小于于或等于 `start`，则不提取任何元素。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:34]:
```suggestion
  - : 提取终止处的索引（从 `0` 开始）, 会[转换为整数](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Number#整数转换). `slice()` 会提取到但不包括 `end` 的位置。
    - 如果索引是负数，则从数组末尾开始计算——如果 `end < 0`，则使用 `end + array.length`。
    - 如果 `end < -array.length` 或者省略了 `end`，则使用 `0`。
    - 如果 `end >= array.length` 或者省略了 `end`，则使用 `array.length`，提取所有元素直到末尾。
    - 如果 `end` 在标准化后小于或等于 `start`，则不提取任何元素。
```

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 4]
```diff
@@ -19,24 +21,29 @@ slice(start, end)
...
 
 ### 返回值
 
 一个含有被提取元素的新数组。
 
 ## 描述
 
-`slice` 不会修改原数组，只会返回一个浅复制了原数组中的元素的一个新数组。原数组的元素会按照下述规则拷贝：
+`slice()` 方法是一个[复制方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#copying_methods_and_mutating_methods)。它不会改变 `this`，而是返回一个[浅拷贝](/zh-CN/docs/Glossary/Shallow_copy)，其中包含了原始数组的一部分元素。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:42]:
```suggestion
`slice()` 方法是一个[复制方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#copying_methods_and_mutating_methods)。它不会改变 `this`，而是返回一个[浅拷贝](/zh-CN/docs/Glossary/Shallow_copy)，其中包含了原始数组的一部分元素。
```
```suggestion
`slice()` 方法是一个[复制方法](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#复制方法和修改方法)。它不会改变 `this`，而是返回一个[浅拷贝](/zh-CN/docs/Glossary/Shallow_copy)，其中包含了原始数组的一部分元素。
```

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 5]
```diff
@@ -19,24 +21,29 @@ slice(start, end)
...
 
-- 如果该元素是个对象引用（不是实际的对象），`slice` 会拷贝这个对象引用到新的数组里。两个对象引用都引用了同一个对象。如果被引用的对象发生改变，则新的和原来的数组中的这个元素也会发生改变。
+`slice()` 方法会保留空槽。如果被切片的部分是[稀疏的](/zh-CN/docs/Web/JavaScript/Guide/Indexed_collections#sparse_arrays)，则返回的数组也是稀疏的。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:44]:
```suggestion
`slice()` 方法会保留空槽。如果被切片的部分是[稀疏的](/zh-CN/docs/Web/JavaScript/Guide/Indexed_collections#稀疏数组)，则返回的数组也是稀疏的。
```

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 6]
```diff
@@ -19,24 +21,29 @@ slice(start, end)
...
 
-- 对于字符串、数字及布尔值来说（不是 {{jsxref("Global_Objects/String", "String")}}、{{jsxref("Global_Objects/Number", "Number")}} 或者 {{jsxref("Boolean")}} 对象），`slice` 会拷贝这些值到新的数组里。在别的数组里修改这些字符串或数字或是布尔值，将不会影响另一个数组。
-
-如果向两个数组任一中添加了新元素，则另一个不会受到影响。
+`slice()` 方法是[通用的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#generic_array_methods)。它只要求 `this` 上有 `length` 属性和整数键。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:46]:
```suggestion
`slice()` 方法是[通用的](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Array#通用数组方法)。它只要求 `this` 上有 `length` 属性和整数键。
```

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 7]
```diff
@@ -46,14 +53,10 @@ slice(start, end)
...
+// citrus 包含 ['Orange','Lemon']
 ```
 
-> **标注：**
->
-> **标注：** citrus _\[n.]_ 柑橘类果实——译注
-
 ### 使用 `slice`
 
 在下例中，`slice` 从 `myCar` 中创建了一个新数组`newCar`。两个数组都包含了一个 `myHonda` 对象的引用。当 `myHonda` 的 `color` 属性改变为 `purple`，则两个数组中的对应元素都会随之改变。
```
Comment (JasonLamv-t) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:62]:
```suggestion
在下例中，`slice` 从 `myCar` 中创建了一个新数组 `newCar`。两个数组都包含了一个 `myHonda` 对象的引用。当 `myHonda` 的 `color` 属性改变为 `purple`，则两个数组中的对应元素都会随之改变。
```

**Review by JasonLamv-t — COMMENTED** [review]
Make some suggestions on formatting issues.

**Comment by JasonLamv-t** [comment]
> > Thanks for your contribution! I already add this PR to the list.
> 
> Thank you. But I translated slice instead of splice

Oops! My fault. I will correct it.

File: files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md [hunk 8]
```diff
@@ -5,9 +5,9 @@ slug: Web/JavaScript/Reference/Global_Objects/Array/slice
 
 {{JSRef}}
 
-**`slice()`** 方法返回一个新的数组对象，这一对象是一个由 `begin` 和 `end` 决定的原数组的**浅拷贝**（包括 `begin`，不包括`end`）。原始数组不会被改变。
+**`slice()`** 方法返回一个新的数组对象，这一对象是一个由 `start` 和 `end` 决定的原数组的[浅拷贝](/zh-CN/docs/Glossary/Shallow_copy)（包括 `start`，不包括 `end`），其中 `start` 和 `end` 代表了数组的下标。原始数组不会被改变。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/array/slice/index.md:8]:
```suggestion
**`slice()`** 方法返回一个新的数组对象，这一对象是一个由 `start` 和 `end` 决定的原数组的[浅拷贝](/zh-CN/docs/Glossary/Shallow_copy)（包括 `start`，不包括 `end`），其中 `start` 和 `end` 代表了数组元素的索引。原始数组不会被改变。
```

**Review by yin1999 — APPROVED** [review]
Thanks. Welcome to the community 🎉

**Pull request merged**